modules/ipi-install-modifying-install-config-for-dual-stack-network.adoc

```diff
@@ -41,3 +44,20 @@ platform:
       - <wildcard_ipv4>
       - <wildcard_ipv6>
 ----
+
+ifdef::vSphere[]
+[IMPORTANT]
+====
+You can configure dual-stack networking on single interface only.
+====
+
+[NOTE]
+====
+* In a vSphere cluster configured for dual-stack networking, the node custom resource object has only the IPv4 address listed in `Status.addresses` field.
+* In the pod that uses the host networking with dual-stack connectivity, the `Status.IP` field contains only the IPv4 address.
```

Comment:
the field is `PodIPs` and `PodIp` not `IP`.  


https://github.com/openshift/origin/blob/release-4.13/vendor/k8s.io/api/core/v1/types.go#L4049

```go
	// IP address of the host to which the pod is assigned. Empty if not yet scheduled.
	// +optional
	HostIP string `json:"hostIP,omitempty" protobuf:"bytes,5,opt,name=hostIP"`
	// IP address allocated to the pod. Routable at least within the cluster.
	// Empty if not yet allocated.
	// +optional
	PodIP string `json:"podIP,omitempty" protobuf:"bytes,6,opt,name=podIP"`

	// podIPs holds the IP addresses allocated to the pod. If this field is specified, the 0th entry must
	// match the podIP field. Pods may be allocated at most 1 value for each of IPv4 and IPv6. This list
	// is empty if no IPs have been allocated yet.
	// +optional
	// +patchStrategy=merge
	// +patchMergeKey=ip
	PodIPs []PodIP `json:"podIPs,omitempty" protobuf:"bytes,12,rep,name=podIPs" patchStrategy:"merge" patchMergeKey:"ip"`
```

Reply:
`namespaces/openshift-cloud-controller-manager-operator/core/pods.yaml`

```yaml
    hostNetwork: true
...
  status:
     hostIP: 192.168.110.69
     phase: Running
     podIP: 192.168.110.69
     podIPs:
     - ip: 192.168.110.69
```

Reply:
Maybe:

In pods that use the host networking with dual-stack connectivity, the `Status.PodIPs` and `Status.PodIP` 
fields contain only the IPv4 address.

Reply:
Sorry folks for coming so late to the party but I have just realized that in this release we have also enabled primary-v6 in dual-stack setups and vSphere also gets this functionality (even though we are not seriously testing it from what I am aware)...

So could we, for the sake of correctness, make both bullets here more general and instead of saying "has only the IPv4 address listed" say "has only the IP address from the primary network listed" ?

This is because for someone doing primary-v6 dual-stack on vSphere we may end up in a situation where docs say "you should see IPv4" and the user sees IPv6 (still, only one entry, not both).

Reply:
@rbbratta @mkowalski Thanks for your review comments. I have incorporated them. Can you review again and let me know if it looks fine?

Reply:
Looks good, thank you! Let's wait for Ross to get the final approval

Reply:
@mkowalski IPv6 Primary was not enabled for vSphere

```go
		case p.BareMetal != nil:
			// We now support ipv6-primary dual stack on baremetal
			allowV6Primary = true
		case p.VSphere != nil:
		case p.OpenStack != nil:
		case p.Ovirt != nil:
		case p.Nutanix != nil:
		case p.None != nil:
		default:
```
https://github.com/openshift/installer/blob/release-4.13/pkg/types/validation/installconfig.go#L241